py/desispec/pixgroup.py

```diff
@@ -122,14 +123,7 @@ def get_exp2healpix_map(survey=None, program=None, expfile=None,
         night = exptab['NIGHT'][i]
         expid = exptab['EXPID'][i]
         tileid = exptab['TILEID'][i]
-        tilepixfile = io.findfile('tilepix', night, expid, tile=tileid)
-
-        if not os.path.exists(tilepixfile):
-            if strict:
-                raise FileNotFoundError(tilepixfile)
-            else:
-                continue
-
+        tilepixfile = io.findfile('tilepix', night, expid, tile=tileid, readonly=True)
```

Comment:
The original `strict=False` option to allow missing tilepix files was to support testing on productions that didn't yet have tilepix files for some nights.  Since the tilepix files should now be created by any new processing, I wanted to simplify the code by removing checks for the possibility that a required file might not be there.  i.e. if it is missing, something really is wrong and we should just crash with a traceback.